packages/http-client-js/src/components/transforms/content-type-encoding-provider.tsx

```diff
@@ -15,7 +15,7 @@ export function ContentTypeEncodingProvider(props: ContentTypeEncodingProviderPr
   };
 
   // Follows the media type conventions from RFC 6838 section 4.2.8, which standardizes the +json suffix for JSON-structured syntaxes.
-  if (contentType?.includes("application/json") || contentType.trim()?.endsWith("+json")) {
+  if (isJsonMediaType(contentType)) {
```

Comment:
the previous logic is problematic for application/jsonl.

Reply:
makes sense thanks!